src/sentry/integrations/bitbucket_server/client.py

```diff
@@ -40,7 +40,8 @@ class BitbucketServerSetupClient(ApiClient):
     authorize_url = "{}/plugins/servlet/oauth/authorize?oauth_token={}"
     integration_name = "bitbucket_server_setup"
 
-    def __init__(self, base_url, consumer_key, private_key, verify_ssl=True):
+    def __init__(self, base_url, consumer_key, private_key, verify_ssl=True, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
I don't know that this will help for bitbucket server - I don't see this initialized with an integration id and the parent class doesn't have it either.